src/main/resources/static/js/main.js

```diff
@@ -0,0 +1,7 @@
+function checkPasswordMatch(fieldConfirmPassword) {
+    if (fieldConfirmPassword.value !== $("#password").val()) {
+        fieldConfirmPassword.setCustomValidity("Passwords do not match!");
+    } else {
+        fieldConfirmPassword.setCustomValidity("");
+    }
+}
```

Comment:
Add newlines at end of new .js and .html files

Reply:
Already done

Reply:
👍